app/utils/application-state-utils.ts

```diff
@@ -598,9 +599,16 @@ async function getAppealApplicationNextStep(req: Request) {
         };
       } else if (ftpaEnabled && APPLICANT_TYPE.RESPONDENT === ftpaApplicantType) {
         const ftpaDecision = req.session.appeal.ftpaRespondentDecisionOutcomeType || req.session.appeal.ftpaRespondentRjDecisionOutcomeType;
-        doThisNextSection = {
-          descriptionParagraphs: i18n.pages.overviewPage.doThisNext.ftpaDecided.respondent[ftpaDecision]
-        };
+        if (ftpaSetAsideFeatureEnabled) {
```

Comment:
have this file some test associated?

Reply:
@MiguelMartinRojas Thanks I'm adding a test scenario in this file now